docs/rfc/0002_sbombastic_version_strategy.md

```diff
@@ -0,0 +1,157 @@
+Finally, the `appVersion` attribute will always be set to the `<Major>.<Minor>.<Patch>` stick to the version of the SBOMBastic stack.
+
+> See the official documentation about
+> [`Chart.yaml`](https://helm.sh/docs/topics/charts/#the-chartyaml-file)
+> for more information.
+
+## Example
+This section outlines scenarios to illustrate how the proposal would work in practice.
+
+### A new release SBOMBastic takes place
+A new version of SBOMBastic stack has to be released because new features has been introduced.
+Assumptions:
+- The current version of the SBOMBastic stack is `1.2.0`
+- The current version of the helm chart is `v1.5.3`
+
+Actions:
+All core components (Controller, Worker, Storage) will be tagged and released as `1.2.0`.
+
+Helm Chart Changes:
+- The chart `version` attribute receives a minor bump too because the version
+  of the Kubewarden stack was bumped: `v1.6.0`
+- The `appVersion` attribute is set to `1.2.0`, because all of the components have the same version.
+
+
+### A patch for SBOMBastic stack
+A patch release is made to deliver backward-compatible bug fixes.
+
+Assumptions:
+- The current version of the SBOMBastic stack is `1.2.0`
+- The current version of the helm chart is `v1.5.3`
+
+Actions:
+All core components (Controller, Worker, Storage) will be tagged and released as `1.2.1`.
+
+Helm Chart Changes:
+- The chart `version` attribute receives a minor bump too because the version
+  of the Kubewarden stack was bumped: `v1.5.4`
+- The `appVersion` attribute is set to `1.2.1`, because all of the components have the same version.
+
+### A patch for helm chart
+Sometimes only the Helm chart needs to be updated, such as adjusting argument defaults or template logic.
+
+Assumptions
+- The current version of the Helm chart is `v1.5.4`
+
+A patch to upgrade Helm chart due to the argument setting.
+The Helm chars will update to `v1.5.5`
+
+### A bug found in SBOMBastic stack
+When users encounter a bug, they can simply report the SBOMbastic stack version they are using (e.g., `1.2.1`).
+This unified versioning approach makes it much easier for maintainers to reproduce the issue and verify the environment across components.
+
+
+# Drawbacks
+
+[drawbacks]: #drawbacks
+
+Every pathc update triggers a full release
+- Even if only a single component is updated, all components must be released together.
+- This comsume unnecessary CI/CD resources and increases operational overhead.
+
+All components must upgrade together
+- Updating the version of one component requires all others to adopt the same version, even if they haven’t changed./
+
+Version numbers may not accurately reflect code changes
+- For example, if the controller is patched multiple times and its version bumps to 1.2.5, the worker and storage components must also be released as 1.2.5, despite having no actual code changes.
+
+# Alternatives
+
+[alternatives]: #alternatives
```

Comment:
Should we say we considered releasing the components separately and explain why we didn’t?